misc/python/genapi.py

```diff
@@ -40,6 +40,11 @@
     #'mesh send',
 ]
 
+# API Version:
+VERSION = '2.0a1'
+# minimega version (set this before rendering)
+MM_VERSION = 'UNKNOWN'
```

Comment:
I'm confused - is the goal to have the generated binding warn/nag if there's a version mismatch? If so, wouldn't you want per commit matching, since we currently do have breaking changes pretty much weekly?